emigration.lua

```diff
@@ -1,8 +1,8 @@
 --Allow stressed dwarves to emigrate from the fortress
+-- Updated for 0.47.05 by wsfsbvchr
```

Comment:
```suggestion
```

Reply:
removing this line as discussed in the other review. Please do add your name to Authors.rst, though!